site/src/components/components/LivePreview.module.css

```diff
@@ -1,5 +1,6 @@
 .container {
   border: var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-primary-borderColor);
+  margin-top: calc(var(--salt-size-unit) * 2);
```

Comment:
maybe we can use the spacing foundation here?

Reply:
we can only use those vars once we upgrade mosaic to the latest :( https://github.com/jpmorganchase/salt-ds/pull/1670